test/thermo/consistency.cpp

```diff
@@ -345,6 +349,56 @@ TEST_P(TestConsistency, dSdv_const_T_eq_dPdT_const_V) {
     }
 }
 
+TEST_P(TestConsistency, betaT_eq_minus_dmv_dP_const_T_div_mv)
+{
+    double betaT1;
+    try {
+        betaT1 = phase->isothermalCompressibility();
+    } catch (NotImplementedError& err) {
+        GTEST_SKIP() << err.getMethod() << " threw NotImplementedError";
+    }
+
+    double T = phase->temperature();
+    double P1 = phase->pressure();
+    double mv1 = phase->molarVolume();
+    
+    double P2 = P1 * (1 + 1e-7);
```

Comment:
If you use the same relative perturbation of 1e-6 that is used on other finite difference tests, I think you can set much tighter absolute tolerances for these tests. At least locally, with this larger perturbation, I was able to pass the test suite with `atol_c = 1e-14` and `atol_e = 1e-18`.

Reply:
Adjust tolerances in a9b594528dbfc1c13cdf63c24abf16e5b4f38254  as mentioned